src/utils/config.ts

```diff
@@ -65,6 +65,7 @@ export const defaultUserConfig: UserConfig = {
   skipLabels: ['skip-release', 'skip-changelog', 'regression'],
   skipCommitsWithoutPullRequest: true,
   commentOnReleasedPullRequests: true,
+  sortTags: true,
```

Comment:
@6543 Why is there an option to disable this?

Reply:
if you want the old behavior instead ... you could do so

Reply:
@anbraten should I drop the config option and make it default?

Reply:
I think its kind of a bugfix and I can't think why someone would want the other behavior, so I would just remove it.